homeassistant/components/homekit/translations/en.json

```diff
@@ -21,7 +22,7 @@
 		  },
 		  "advanced" : {
 			 "data" : {
-				"auto_start" : "Autostart (disable if using Z-Wave or other delayed start system)",
+				"auto_start" : "[%key:component::homekit::config::step::user::data::auto_start%]",
 				"safe_mode" : "Safe Mode (enable only if pairing fails)",
```

Comment:
Don't forget to use references for the other ones

Reply:
found another one :)